app/src/main/java/com/hedvig/app/feature/insurance/ui/detail/yourinfo/YourInfoFragment.kt

```diff
@@ -31,44 +31,57 @@ class YourInfoFragment : Fragment(R.layout.contract_detail_your_info_fragment) {
                     (adapter as? YourInfoAdapter)?.submitList(
                         homeSection(
                             it.address.fragments.addressFragment,
-                            it.squareMeters,
-                            it.saType.displayName(requireContext())
-                        ) + coinsuredSection(it.numberCoInsured)
+                            it.saType.displayName(requireContext()),
+                            it.squareMeters
+                        ) + coinsuredSection(it.numberCoInsured) +
+                            YourInfoModel.Paragraph(
+                                getString(R.string.insurance_details_view_your_info_co_insured_footer)
+                            ) + YourInfoModel.Header(
```

Comment:
This header, paragraph and button appears to be a logical 'section', perhaps we could extract it into a function as well?